src/frontend/staff-portal/src/app/components/jj-workbench/jj-dispute-digital-case-file/jj-dispute-digital-case-file.component.ts

```diff
@@ -56,16 +59,14 @@ export class JJDisputeDigitalCaseFileComponent implements OnInit, AfterViewInit
 
   applyFilter() {
     this.dataSource.filter = this.filterText;
+    this.tableHeight = this.calcTableHeight(300);
   }
 
   refreshData(jjDisputes: JJDispute[]): void {
     this.data = jjDisputes;
```

Comment:
Oops this can be removed too, I will approve this PR, you can merge after change